### PR TITLE
NMA-1262  Polishing

### PR DESCRIPTION
**Pull request by hadia**
UI changes to polish CrowdNode screens

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: common/src/main/res/layout/dialog_adaptive.xml [hunk 1]
```diff
@@ -37,7 +37,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="20dp"
         android:gravity="center"
-        android:textAlignment="gravity"
+        android:layout_gravity="center"
+        android:textAlignment="center"
```
Comment (Syn-McJ) [inline common/src/main/res/layout/dialog_adaptive.xml:41]: This breaks the logic of aligning a message that is longer than 3 lines to the left (right in rtl locales). Is there a reason for this change?

**Review by Syn-McJ — COMMENTED** [review]
@hadia I dodn't see the changes that are related to some ACs like

- New Account Screen
    - Remove “in”
   -  “To and from CrowdNode”

Both are still there: 
`You will lose access to your funds in within this wallet...`
`All transfers to and from CrowdNode from...`

File: common/src/main/res/layout/dialog_adaptive.xml (hunk 1 above)
Comment (hadia) [reply common/src/main/res/layout/dialog_adaptive.xml:41]: @Syn-McJ I tried to match new figma design 
image

Comment (Syn-McJ) [reply common/src/main/res/layout/dialog_adaptive.xml:41]: You should be able to achieve that just by removing the icon though? Changing alignment logic isn't necessary.

**Pull request merged**